tox.ini

```diff
@@ -6,11 +6,14 @@ skipsdist = True
 [testenv]
 deps = -r{toxinidir}/requirements.txt
        -r{toxinidir}/test-requirements.txt
+install_command = pip install {opts} {packages}
 
 [testenv:linters]
 commands =
   molecule lint
 
 [testenv:molecule]
+setenv =
+  MOLECULE_NO_LOG=false
```

Comment:
This is no longer needed because of this PR: https://github.com/ansible-security/ids_config/pull/10